Lib/glyphsLib/builder/axes.py

```diff
@@ -165,12 +165,16 @@ def to_designspace_axes(self):
                 continue
         # See https://github.com/googlefonts/glyphsLib/issues/280
         elif font_uses_axis_locations(self.font):
-            # Build the mapping from the "Axis Location" of the masters and instances
+            # If all masters have an "Axis Location" custom parameter, only the values
+            # from this parameter will be used to build the mapping of the masters and instances
             # TODO: (jany) use Virtual Masters as well?
+            #       (jenskutilek) virtual masters can't have an Axis Location parameter.
             mapping = {}
             for master in self.font.masters:
                 designLoc = axis_def.get_design_loc(master)
-                userLoc = axis_def.get_user_loc(master)
```

Comment:
hm wait a sec.. are you now completely byassing `get_user_loc` method that would previously do the guessing based on weight class and only considering Axis Location? I think before the latter would override implicit weight/width class settings if not None, but if None those implicit settings would still count.. now it seems you simply skip them in the None case..

Reply:
but it may be present while also omit some axis coordinates, can it not? which is why you are checking for None.. whereas before it was never None, because even if an axis was omitted from an Axis Location parameter, there would still be a userLoc defined (inferred from other values)

Reply:
> but it may be present while also omit some axis coordinates, can it not? which is why you are checking for None.. whereas before it was never None, because even if an axis was omitted from an Axis Location parameter, there would still be a userLoc defined (inferred from other values)

I haven't checked in detail, but it seems that via the UI in Glyphs it is impossible to omit coordinates for some axes.

Reply:
If the Axis Location parameter is disabled in a master or instance, it will return `None`. I’ve updated my code accordingly.